src/main/java/network/brightspots/rcv/CandidateRankingsList.java

```diff
@@ -81,6 +81,9 @@ CandidatesAtRanking get(int i) {
   }
 
   int maxRankingNumber() {
+    if (numRankings == 0) {
+      throw new IllegalArgumentException("Max ranking may only be called on non-empty rankings");
```

Comment:
Is adding this actually necessary? Just concerned that it might cause an exception when one isn't warranted (e.g. in cases when `maxRanking` isn't defined and therefore defaults to 0).